README.md

```diff
@@ -15,6 +15,13 @@ When using drush on the host you need:
 ```
 export IS_DDEV_PROJECT=true
 ```
+or this in your `settings.local.php`:
```

Comment:
(Need to add a note about enabling settings.local.php in settings.php?)

Reply:
Ah, but please use DDEV instead of using PHP on the host :) If I had my way people would uninstall PHP from their computers in order to not have all the confusion that comes with it.